pkg/controllers/deprovisioning/controller.go

```diff
@@ -172,7 +172,7 @@ func (c *Controller) deprovision(ctx context.Context, deprovisioner Deprovisione
 }
 
 func (c *Controller) executeCommand(ctx context.Context, d Deprovisioner, command Command) error {
-	deprovisioningActionsPerformedCounter.WithLabelValues(fmt.Sprintf("%s/%s", d, command.Action())).Add(1)
+	deprovisioningActionsPerformedCounter.WithLabelValues(string(command.Action()), d.String()).Add(1)
```

Comment:
Can we use `With` instead of `WithLabelValues`? I think in general, we've found that it's a bit clearer what each label maps to if you do this.

Reply:
`WithLabelValues` definitely works better as a one-liner :) But yeah I think obscuring the label key is a little costly from a readability/maintainability perspective. So a multi-line `prometheus.Labels` struct literal inside a `With()` is my preferred way to read these, fwiw

Reply:
Good point, I can add that in.